examples/veymont-global-programs/TicTacToe.pvl

```diff
@@ -1,6 +1,5 @@
 // -*- tab-width:4 ; indent-tabs-mode:nil -*-
-//:: cases TicTacToeInt TicTacToeMatrix
-//:: tools veymont
```

Comment:
why did you remove this line?

Reply:
The global programs are run with the script in the util folder, so they do not have to be included in vercor's own test suite mechanism (the one triggered by passing the flag ```--test=./examples```). Removing the "tools" directive ensures it's not included in the test suite.

Reply:
The script is at the bottom of the "files changes" page of this PR, it's called `veymont-2stage-tests.sh`.

Reply:
yes sorry, I realised that it might be that but did not remove the comment, the thing is that as you mention the script was at the bottom :D